java/opendcs/src/main/java/decodes/tsdb/comprungui/RunComputationsFrameTester.java

```diff
@@ -1,7 +1,14 @@
 package decodes.tsdb.comprungui;
```

Comment:
I know this existed already but would it make sense to just add the functionality to the main CompRunGuiFrame class?

Reply:
To my knowledge(which I may be wrong about) is that runComputationsFrameTester extends TsdbAppTemplate, where CompRunGuiFrame extends TopFrame. So CompRunGuiFrame does not have the capability to be run is a stand alone mode, so it needs the TsdbAppTemplate to initialized the gui then open it.